docs/app/docs/styling/[slug]/components/text-gradient/static.tsx

```diff
@@ -0,0 +1,42 @@
+import { cerberus } from '@cerberus-design/react'
+
+export default function TextGradientPreview() {
+  return (
+    <cerberus.div
+      display="flex"
+      flexDirection="column"
+      gap="4"
+      padding="6"
+      borderRadius="lg"
+      border="1px solid"
```

Comment:
We don't need a border around this div. Let's get rid of it so the preview is cleaner.